include/proto/plugin_pr_curve.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package mlboard;
+
+message PrCurvePluginData {
+  // Version `0` is the only supported version.
+  int32 version = 1;
```

Comment:
I'm confused, if 0 is the only supported version we still use 1 here, or is that the upper bound?

Reply:
This was there in the older version, they changed it, I removed the comment

Reply:
https://github.com/tensorflow/tensorboard/blob/7511300d6ca5d25d332459d89e5725a47baf225b/tensorboard/plugins/pr_curve/plugin_data.proto#L21

Reply:
The comments are still there but anyways